example/pkg/apis/miskatonic/v1beta1/university_types.go

```diff
@@ -26,7 +26,7 @@ import (
 // +genclient=true
 
 // +k8s:openapi-gen=true
-// +resource=universities
```

Comment:
Sorry I was unclear on slack.  Please create a new type under miskatonic/v1beta1 in a new `yourtype_types.go` file with a new `yourtype_types_test.go` test.